site/src/modules/resources/AppLink/AppLink.tsx

```diff
@@ -119,12 +119,11 @@ export const AppLink: FC<AppLinkProps> = ({ app, workspace, agent }) => {
         endIcon={isPrivateApp ? undefined : <ShareIcon app={app} />}
         disabled={!canClick}
         href={href}
-        target="_blank"
         css={{
           pointerEvents: canClick ? undefined : "none",
           textDecoration: "none !important",
         }}
-        onClick={async (event) => {
+        onClick={async (event: MouseEvent<HTMLElement>) => {
```

Comment:
Tried seeing what would happen if I removed the type annotation, and man, MUI really is a disaster of megamorphic abstractions, huh

Reply:
yes, yes it is :^)